client/components/domains/featured-domain-suggestions/styles/base.scss

```diff
@@ -85,9 +93,45 @@
 	}
 }
 
-.featured-domain-suggestions--has-match-reasons .featured-domain-suggestion {
-	.domain-registration-suggestion__match-reasons {
-		order: 3;
-		flex-basis: 26px;
+@include breakpoint( '<660px' ) {
+	.featured-domain-suggestions {
+		flex-flow: wrap;
+
+		.featured-domain-suggestion .domain-suggestion__action.button.is-primary {
+			width: initial;
+		}
+	}
+}
+
+@include breakpoint( '>660px' ) {
+	.featured-domain-suggestions {
+		.domain-registration-suggestion__title {
+			.featured-domain-suggestions--title-in-18em & {
+				font-size: 1.8em;
+			}
+			.featured-domain-suggestions--title-in-16em & {
+				font-size: 1.6em;
+			}
+			.featured-domain-suggestions--title-in-14em & {
+				font-size: 1.4em;
+			}
+			.featured-domain-suggestions--title-in-12em & {
+				font-size: 1.2em;
+			}
+			.featured-domain-suggestions--title-in-10em & {
+				font-size: 1em;
+			}
+		}
+
+		.featured-domain-suggestions--title-causes-overflow {
```

Comment:
I think this should be `&.featured-domain-suggestions--title-causes-overflow`.

The way this is implemented prevents these styles from being applied when the title causes an overflow.

<img width="999" alt="wordpress_com" src="https://user-images.githubusercontent.com/1379730/39433657-6722ff0e-4c64-11e8-94b0-e08e74101d33.png">

Reply:
D'oh! I'll look into this.